python/lsst/verify/measurement.py

```diff
@@ -387,7 +392,9 @@ def deserialize(cls, metric=None, identifier=None, value=None, unit=None,
         else:
             _blobs = None
 
-        # Resolve quantity
+        # Resolve quantity, represent None values as np.nan
+        if value is None:
+            value = np.nan
```

Comment:
This means that values that were `inf` originally will be mapped to `np.nan` going forward.  I guess there is nothing we can really do about that.